examples/multi_gate.rst

```diff
@@ -178,7 +178,7 @@ those that fail to do so.
     # Check how many solutions meet the constraints (are valid)
     valid, invalid, data = 0, 0, []
     for datum in response.data():
-        sample, energy, num = datum
+        sample, energy, num, _chain_break_fraction = datum
         if (csp.check(sample)):
```

Comment:
For today's urgent update to virtualgraphs I also snuck in a change to update for the updated dimod.response. So now this proposed change has a conflict